core/modules/system/system.admin.inc

```diff
@@ -1798,10 +1798,13 @@ function system_jquery_settings() {
     '#title' => t('jQuery version'),
     '#default_value' => $config->get('jquery_version'),
     '#options' => $options,
-    '#description' => t('<a href="@url">More information about jQuery</a> can be found in the documentation.', array('@url' => url('https://docs.backdropcms.org/documentation/introducing-jquery'))),
   );
   $form['jquery_version']['default']['#description'] = t('The latest, default version of jQuery.');
   $form['jquery_version']['legacy']['#description'] = t('Use the legacy version if any modules or themes are not working properly with the latest version.');
+  $form['help'] = array(
+    '#type' => 'help',
+    '#markup' => t('<a href="@url">More information about jQuery</a> can be found in the documentation.', array('@url' => url('https://docs.backdropcms.org/documentation/jquery'))),
+  );
```

Comment:
Sorry @herbdool, I should have been more specific about this. The UX issue here was that the help text of the last radio item was too close with the help text for the entire radio set element. What I meant was for this new help element to come before the radio set, at the top of the page/form.

Reply:
...actually never mind. I don't want to hold this up.